data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,851 @@
+		"hardpoint offset" 8
+		sound "astuit"
+		"hit effect" "astuit hit"
+		"inaccuracy" 5
+		"turret turn" 2.2
+		"velocity" 8
+		"lifetime" 44
+		"reload" 6
+		"firing energy" 13.2
+		"firing heat" 34
+		"shield damage" 8.8
+		"hull damage" 9.4
+	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat, although the energy requirement of such technology is still significant. Astuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With four blasters on a turreted mount, it produces a constant stream of damage which adds up over time, despite the minor damage that each individual projectile creates."
+
+effect "astuit hit"
+	sprite "effect/ast hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 6
+	"velocity scale" .001
+
+
+outfit "Acuit Artillery"
+	plural "Acuit Artillery"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 1406000
+	thumbnail "outfit/acuit"
+	"mass" 81
+	"outfit space" -81
+	"weapon capacity" -81
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/acuit"
+			"random start frame"
+		"hardpoint sprite" "hardpoint/acuit"
+		"hardpoint offset" 14
+		"hit effect" "acuit hit"
+		sound "acuit"
+		"inaccuracy" 3
+		"turret turn" 1.3
+		"velocity" 18
+		"lifetime" 60
+		"reload" 56
+		"burst reload" 10
+		"burst count" 1
+		"firing energy" 130
+		"firing heat" 204
+		"shield damage" 170
+		"hull damage" 290
+		"hit force" 40
+	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. Acuit Artillery outranges most weapons in Gegno space, while having a higher rate of fire as well, making them very efficient bombardment weapons.`
+
+effect "acuit hit"
+	sprite "effect/acuit hit"
+		"frame rate" 18
+		"no repeat"
+	"lifetime" 18
+	"velocity scale" -0.004
+
+
+outfit "Guile Pulse Laser"
+	category "Guns"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 109000
+	"thumbnail" "outfit/guile pulse laser"
+	"mass" 15
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	weapon
+		"sprite" "projectile/guile pulse laser"
+			"frame rate" 6
+		sound "guile pulse"
+		"hit effect" "heavy laser impact"
+		"inaccuracy" 0.2
+		"velocity" 150
+		"lifetime" 1
+		"reload" 6
+		"burst count" 3
+		"burst reload" 1
+		"firing energy" 3.7
+		"firing heat" 10
+		"shield damage" 12.4
+		"hull damage" 20.2
+		"cluster"
+	description `The Guile Pulse Laser is a recent experimental advancement in Scin plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 
```

Comment:
To be honest, this sounds like what Heavy Lasers should be. Also reminds me of the "Thunderhead lance" from EVN, which is a good thing. Thunderhead lances were beams the way beams should be used in human-tech-level areas. Very powerful, but with significant downsides, particularly range. They were daggers in a gunfight, but daggers that were *highly* lethal if they could get close enough to be used.

Reply:
Ironically but unintentionally, Halites also *look* like Thunderheads, so I find it fun you draw that similarity as well. I suppose the Heavy laser issue is due to the name not including beam. These are really pulse lasers, and heavy lasers are beams, which separate them.

Reply:
That makes sense. The more I play in my "full playthrough," the less I like the lasers in human space. Too good as a regular "every ship can make good use of them" kind of weapon. Whereas the thunderhead was a "great if you can get into close quarters, but a poor choice for most ships" weapon.